docs/ref/settings.txt

```diff
@@ -3032,6 +3032,10 @@ The domain to use for session cookies. Set this to a string such as
 ``"example.com"`` for cross-domain cookies, or use ``None`` for a standard
 domain cookie.
 
+Equivalent to :setting:`ALLOWED_HOSTS`, a value beginning with a period can be
+used as a subdomain wildcard: '.example.com' will match example.com,
+www.example.com, and any other subdomain of example.com.
```

Comment:
I don't think that we should mention `ALLOWED_HOSTS`, maybe a small rewording e.g.:
```
...
domain cookie. The setting also supports subdomains, so you could use
``".example.com"``, for example, to allow access from all subdomains of
``example.com``.
```

Reply:
Thanks, used your text. 👍